src/System.Runtime.Extensions/src/System/IO/BufferedStream.cs

```diff
@@ -1246,6 +1246,30 @@ public override void SetLength(long value)
             _stream.SetLength(value);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
```

Comment:
The implementation here is largely copied from the `CopyToAsync` impl, with the inner `CopyToAsyncCore` method eliminated and written inline.